pass/pass_linux.go

```diff
@@ -20,29 +20,13 @@ import (
 const PASS_FOLDER = "docker-credential-helpers"
 
 var (
-	PassInitialized bool
+	passInitialized bool
 )
 
-func init() {
-	// In principle, we could just run `pass init`. However, pass has a bug
-	// where if gpg fails, it doesn't always exit 1. Additionally, pass
-	// uses gpg2, but gpg is the default, which may be confusing. So let's
-	// just explictily check that pass actually can store and retreive a
-	// password.
-	password := "pass is initialized"
-	name := path.Join(PASS_FOLDER, "docker-pass-initialized-check")
-
-	_, err := runPass(password, "insert", "-f", "-m", name)
-	if err != nil {
-		return
-	}
-
-	stored, err := runPass("", "show", name)
-	PassInitialized = err == nil && stored == password
-
-	if PassInitialized {
-		runPass("", "rm", "-rf", name)
-	}
+func IsPresent() bool {
+	_, err := exec.LookPath("pass")
+	fmt.Println(err)
```

Comment:
fmt.Println(err) ?

Reply:
Should we ignore `permission denied` errors etc? (Was thinking if we should print any error other than`ErrNotFound`).

Also wondering if there's a need to cache the result, but possibly better left to the software using this

Reply:
I think we shouldn't bother providing this 2-line helper function. The caller can decide to check if `pass` is installed themselves just fine.

Reply:
@euank I don't agree with that, a helper is meant to be pluggable and should remove the need for the caller to be aware of implementation details.